samples/client/petstore/kotlin-gson/docs/PetApi.md

```diff
@@ -155,7 +155,7 @@ Configure petstore_auth:
 ### HTTP request headers
 
  - **Content-Type**: Not defined
- - **Accept**: application/xml, application/json
```

Comment:
@sheepdreamofandroids is this change expected?

Reply:
Yes, this client doesn't support xml.